src/main.cpp

```diff
@@ -24,6 +25,8 @@ static constexpr auto USAGE =
 
 int main(int argc, const char **argv)
 {
+  fmt::print("Version\nMajor {}\nMinor {}\nPatch {}\nGit Hash {}\n", Version::Major, Version::Minor, Version::Patch, Version::GitHash);
```

Comment:
might be a nice place for a raw string literal with the line breaks in the code?